{{cookiecutter.project_slug}}/README.md

```diff
@@ -72,3 +72,65 @@ To create a superuser use the following commands:
 If React frontend was selected during the project creation
 (using our [cookiecutter](https://github.com/sixfeetup/cookiecutter-sixiedjango)), you
 can access it at [http://localhost:3000/](http://localhost:3000/).
+
+## Install Loki for log aggregation and the Kube Prometheus Stack with Grafana Dashboards
+
+Add a node to your cluster that you want to use for monitoring and run `kubectl
+get nodes` for a list of nodes. You should see output similar to the following:
+
+```
+$ kubectl get nodes
+NAME               STATUS   ROLES    AGE   VERSION
+pool-fc410-bivim   Ready    <none>   48d   v1.26.3
+pool-fc410-lziru   Ready    <none>   48d   v1.26.3
+pool-fc410-qdznj   Ready    <none>   15d   v1.26.4
+```
+
+Pick the node that you want to use for monitoring and add the
+"nodetype=monitoring" label to it:
+
+```
+kubectl label nodes pool-fc410-qdznj nodetype=monitoring
+```
```

Comment:
It is not necessary to set node affinity for the cookie cutter. This can be a per-project decision.